### ISSUE TITLE
when use `podman run -q`,`-q` option doesn't work

### ISSUE THREAD

**Issue by wenj1104** [containers/libpod#2216]
/kind bug

**Description**

When i use `podman run -q` to run a container,the output of pull download should'n be showed.

**Steps to reproduce the issue:**

1. Run a container.
```
# podman run -q -d alpine sleep 10
Trying to pull docker.io/alpine:latest...Getting image source signatures
Copying blob cd784148e348: 2.10 MiB / 2.10 MiB [============================] 4s
Copying config 3f53bb00af94: 1.48 KiB / 1.48 KiB [==========================] 0s
Writing manifest to image destination
Storing signatures
ef48adca4ade0245d360ed80c39c01494748e1b1493bd404d321f9e8da132d26
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**
```
# podman version
Version:       1.0.0
Go Version:    go1.11.4
Git Commit:    "49780a1cf10d572edc4e1ea3b8a8429ce391d47d"
Built:         Tue Jan 15 04:38:17 2019
OS/Arch:       linux/amd64
```

**Output of `podman info`:**

```
# podman info
host:
  BuildahVersion: 1.6-dev
  Conmon:
    package: podman-1.0.0-1.git82e8011.fc29.x86_64
    path: /usr/libexec/podman/conmon
    version: 'conmon version 1.12.0-dev, commit: 49780a1cf10d572edc4e1ea3b8a8429ce391d47d'
  Distribution:
    distribution: fedora
    version: "29"
  MemFree: 6050058240
  MemTotal: 8338890752
  OCIRuntime:
    package: runc-1.0.0-66.dev.gitbbb17ef.fc29.x86_64
    path: /usr/bin/runc
    version: |-
      runc version 1.0.0-rc6+dev
      commit: ead425507b6ba28278ef71ad06582df97f2d5b5f
      spec: 1.0.1-dev
  SwapFree: 4294963200
  SwapTotal: 4294963200
  arch: amd64
  cpus: 8
  hostname: tool.example.com
  kernel: 4.18.16-300.fc29.x86_64
  os: linux
  rootless: false
  uptime: 1h 19m 24.48s (Approximately 0.04 days)
insecure registries:
  registries: []
registries:
  registries:
  - docker.io
  - registry.docekr-cn.com
store:
  ConfigFile: /etc/containers/storage.conf
  ContainerStore:
    number: 7
  GraphDriverName: overlay
  GraphOptions: null
  GraphRoot: /var/lib/containers/storage
  GraphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
  ImageStore:
    number: 12
  RunRoot: /var/run/containers/storage
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
```
# cat /etc/redhat-release
Fedora release 29 (Twenty Nine)

# uname -a
Linux tool.example.com 4.18.16-300.fc29.x86_64 #1 SMP Sat Oct 20 23:24:08 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by TomSweeneyRedHat** [comment]
@wenj1104 just to be sure, the result of the command is OK thought right?  I.e. the run completed successfully, but it was noisier than it should have been.  We'll take a look.

**Comment by wenj1104** [comment]
@TomSweeneyRedHat Sorry for my late.The result of the command was OK,just noisier than it should have been.

**Comment by rhatdan** [comment]
@wenj1104 Could you try out the patch and see if it fixes your issue.

**Comment by mheon** [comment]
Should be closed via #2230
